src/utils/share.ts

```
@@ -6,8 +6,8 @@ export function useShareReciter() {
   const { formatMessage } = useIntl();
 
   async function shareReciter(reciter: Reciter): Promise<void> {
-    const url = `https://tarteel.quran.us.kg/reciter/${reciter.id}?moshafId=${reciter.moshaf.id}`;
-
+    const baseUrl = process.env.NEXT_PUBLIC_APP_URL || 'https://tarteel.quran.us.kg';
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)